index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <main>
+        <header class="header d-flex flex-center">
+            <a href="#">Alejandro Ramos</a>
+            <a class="burger" href="#"><img src="images/IconMenu.png" alt="burger button"></a>   
+        </header>
+        <section class="mainContain d-flex">
+            <div class="informativeText">
+                <h1>Hey There.<br>I'm Alex</h1>
+                <h2>I’am a Software Developer</h2>
+                <p>I can help you build a product, feature or<br>website Look through some of my work<br>and experience! If you like what you see<br>and have a project you need coded,<br> 
+                don’t hestiate to contact me.</p>
+            </div>
+            <div class="socialIcons">
+                <ul class="d-flex">
+                    <li>
+                        <a href="https://github.com/AlexRS90">
+                            <img src="images/IconGitHub.png" alt="-GitHub button">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/alejandro-ramos-santos-9b0b52135/">
+                            <img src="images/IconLinkedin.png" alt="Linkedin button">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://angel.co/">
+                            <img src="images/IconAngelist.png" alt="Angelist button">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://twitter.com/AlejandroRBenji">
+                            <img src="images/IconTwitter.png" alt="Twitter button">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://medium.com/">
+                            <img src="images/IconMedium.png" alt="Medium button">
+                        </a>
+                    </li>
+            </ul>
+        </div>        
+        </section>
+    </main>
```

Comment:
- [ ] **[OPTIONAL]** You seem to have misunderstood the structure of the HTML tags suggested in the previous review.

Right now you have:
```
<main>
  <header></header>
  <section></section>
</main>
```
But what was suggested was this:
```
<header></header>
<main>
  <section></section>
</main>
```

Reply:
😮 😮 
ok ok perfect! Now I got it 😅 

One last question, to submit my repo, which link should be:

 Main repo link or this PR link?

Thanks in advance!

Reply:
You should submit the **main repo link** after the project has been approved. Remember to merge the Pull Request too!